intro_to_python/intro_to_python.md

```diff
@@ -235,23 +249,23 @@ Your computer most likely will **not** have Python already installed. To check,
+If you prefer to continue practicing in the cloud instead of downloading Python, [Google Colaboratory](https://colab.research.google.com/), or Google Colab, provides an online Jupyter notebook environment that is completely free (though you do need a Google account). The introduction is actually written as a Colab notebook itself, so you can jump right in practicing. While it won't look exactly like the Jupyter notebook we used for this module, it will be a very similar experience, with a mix of code cells and text cells written in Markdown.
 
 You can even see what this module's notebook looks like in Google Colab! Once you've worked through the introduction above:
 
 1. Click "File" in the upper left-hand corner and then "Open notebook" in the drop-down menu:
 
-   ![Opening a notebook in Google Colab](media/google_colab_open_nb.png)
+   ![To open a notebook in Google Colab, click the File drop down menu in the top left, and select the second option: Open notebook](media/google_colab_open_nb.png)
 
  2. In the window that appears, one of the options at the top is **GitHub**. Click that, and you'll see where you can type in a GitHub URL.
  3. In that space, copy and paste the repository where these modules live: [https://github.com/arcus/education_modules](https://github.com/arcus/education_modules).
- 4. Click the magnifying glass to the far right. You should see a list of the Python notebooks in that repository.
+ 4. Click the magnifying glass to the far right to search. You should see a list of the Python notebooks in that repository.
  5. When you find the one you want, the button on the far right (an upward-slanting arrow in a box) will open the notebook file as a Colab notebook in another tab.
 
    <div style="display:none">@gifPreload</div>
 
    <figure>
-     <img src="https://github.com/arcus/education_modules/blob/update-intro-to-python/intro_to_python/media/github_repo_colab.png?raw=true" height="500" width="800" alt="Finding a GitHub repository in Google Colab" data-alt="https://github.com/arcus/education_modules/blob/update-intro-to-python/intro_to_python/media/github_repo_colab.gif?raw=true">
+     <img src="https://github.com/arcus/education_modules/blob/update-intro-to-python/intro_to_python/media/github_repo_colab.png?raw=true" height="500" width="800" alt="Finding a GitHub repository in Google Colab following the above instructions." data-alt="https://github.com/arcus/education_modules/blob/update-intro-to-python/intro_to_python/media/github_repo_colab.gif?raw=true">
```

Comment:
this seems like a sufficient compromise between the conflict of blank alt text tag vs knowledge of an image and lack of certainty of if there's any missing info. 

And of course, if we ever get feedback from folks without visual access, we can adjust based on that!